src/components/tabs/index.tsx

```diff
@@ -33,7 +33,8 @@ const Tabs = ({
 	/**
 	 * Track whether tabs are overflowing, so we can switch to a select.
 	 */
-	const [hasOverflow, overflowRef] = useOverflowRef<HTMLDivElement>()
+	const ref = useRef<HTMLDivElement>()
+	const hasOverflow = useHasOverflow<HTMLDivElement>(ref)
```

Comment:
💭 thought / suggestion: IMO, it feels more intuitive to have the hook return a ref for use, as opposed to passing one in, as now the consumers of the hook _need_ to import and declare a ref to use the hook. It would be functionally equivalent to declare the ref inside `useHasOverflow` and return it in a tuple similar to how the previous API looked.

Reply:
@BRKalow Oh! I think I see what you're saying now. I thought the suggestion was to go back to exactly what the API was before. Previously it returned a callback function intended to be passed as an element's `ref` prop, not an actual ref. You're suggesting to return a ref, not a callback like before, is that right?

Reply:
@ashleemboyer Ah word, yeah just returning a ref object, not a ref callback, is just fine!